content/blog/2015-05-01-graphql-introduction.md

```diff
@@ -3,7 +3,7 @@ title: "GraphQL Introduction"
 author: [schrockn]
 ---
 
-At the React.js conference in late January 2015, we revealed our next major technology in the React family: [Relay](http://facebook.github.io/react/blog/2015/02/20/introducing-relay-and-graphql.html). 
+At the React.js conference in late January 2015, we revealed our next major technology in the React family: [Relay](https://reactjs.org/blog/2015/02/20/introducing-relay-and-graphql.html). 
```

Comment:
This should be a relative link instead

Reply:
Thanks for pointing out!

Reply:
Updated the URL which is having relative link